<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerDialog</class>
 <widget class="QDialog" name="registerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create account</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="accountGroupBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>16</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="title">
        <string>Please create an account</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="firstNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lastNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="emailLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>User ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userIdLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>Confirm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="confirmLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>11</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>Birthday</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="selectedDateLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>14</pointsize>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="locale">
               <locale language="English" country="NewZealand"/>
              </property>
              <property name="maximumDateTime">
               <datetime>
                <hour>10</hour>
                <minute>59</minute>
                <second>59</second>
                <year>9996</year>
                <month>12</month>
                <day>31</day>
               </datetime>
              </property>
              <property name="displayFormat">
               <string>dd/MM/yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="gender">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>11</pointsize>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="title">
           <string>Gender</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="maleRadioButton">
               <property name="font">
                <font>
                 <family>Tahoma</family>
                 <pointsize>13</pointsize>
                 <kerning>false</kerning>
                </font>
               </property>
               <property name="text">
                <string>Male</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="femaleRadioButton">
               <property name="font">
                <font>
                 <family>Tahoma</family>
                 <pointsize>13</pointsize>
                 <kerning>false</kerning>
                </font>
               </property>
               <property name="text">
                <string>Female</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="diverseRadioButton">
               <property name="font">
                <font>
                 <family>Tahoma</family>
                 <pointsize>13</pointsize>
                 <kerning>false</kerning>
                </font>
               </property>
               <property name="text">
                <string>Diverse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createAccountButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create account</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="src.qrc">:/images/jab.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>By clicking the&quot;Create account&quot; button, you are creating a My Covid Record account. All personal information will be kept confidential and only be used by the &quot;My Covid Records&quot;app</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
